mbuild/conversion.py

```diff
@@ -743,6 +744,12 @@ def save(compound,
         and geometric combining rules respectively.
     foyer_kwargs : dict, optional, default=None
         Keyword arguments to provide to `foyer.Forcefield.apply`.
+    combine : 'all', None or list iterables, optional, default='all'
+        Specific for parmed GRO writer. User can choose from
+        If None, system atom order may be changed to meet the
+        need for continguously bonded groups of atoms to be part
+        of a single molecule type. All other values leave the
```

Comment:
I am not entirely sure what this means. 

>all other values leave the atom order unchanged.

Do you mean other file extensions? Or other options that are not `all` for the combine flag?

Reply:
Yeah, I kinda copy this from the parmed docs, it supposed to mean all other values other than `all`